packages/account-usage/lib/capping.ts

```diff
@@ -68,7 +68,7 @@ export class Capping {
             const dimensions = {
                 accountId: plan.account_id,
                 dryRun: this.options?.enabled ? 'false' : 'true',
-                ...Object.fromEntries(Object.keys(status.metrics).map((key) => [key, 'true']))
+                ...Object.fromEntries(Object.keys(status.metrics).map((key) => [`metric-${key}`, 'true']))
```

Comment:
unrelated.
adding a prefix so it is easier to differentiate those dimensions with the one automatically added at ingestion (environment, service, etc...)